src/ui/Templates/CoreHelpers.ts

```diff
@@ -664,7 +664,14 @@ TemplateHelpers.registerFieldHelper('anchor', (href: string, options?: IAnchorUt
   return AnchorUtils.buildAnchor(href, options);
 });
 
-TemplateHelpers.registerFieldHelper('image', (src: string, options?: IImageUtilsOptions) => {
+TemplateHelpers.registerFieldHelper('image', (src: string, options?: IImageUtilsOptions, result: IQueryResult = resolveQueryResult()) => {
+  if (options && options.srcTemplate) {
+    return ImageUtils.buildImage(StringUtils.buildStringTemplateFromResult(options.srcTemplate, result), {
```

Comment:
I'm thinking you could probably just modify/adapt/use the component `ImageFieldValue` component, which is a wrapper around this thing here, and makes it much easier to use in templates and document cleanly.

Instead of having to deal with cryptic configuration with a `FieldValue` component + `helper` options.

Reply:
Got it, I will take a look at that. Didn't know it existed.